src/ray/raylet/raylet_client.h

```diff
@@ -0,0 +1,180 @@
+#ifndef RAYLET_CLIENT_H
+#define RAYLET_CLIENT_H
+
+#include <unistd.h>
+#include <mutex>
+#include <unordered_map>
+#include <vector>
+
+#include "ray/raylet/task_spec.h"
+#include "ray/status.h"
+
+using ray::ActorID;
+using ray::JobID;
+using ray::ObjectID;
+using ray::TaskID;
+using ray::UniqueID;
+
+using MessageType = ray::protocol::MessageType;
+using ResourceMappingType =
+    std::unordered_map<std::string, std::vector<std::pair<int64_t, double>>>;
+
+class RayletConnection {
+ public:
+  /// Connect to the local scheduler.
```

Comment:
local scheduler  -> raylet

Reply:
there are also a few other places in this file that still refers to local scheduler